platforms/documentation/docs/src/snippets/buildlifecycle/taskExecutionEvents/tests-kotlin/taskExecutionEvents.kotlin.sample.conf

```diff
@@ -3,7 +3,8 @@
 # end::cli[]
 executable: gradle
 args: broken
-flags: --quiet
+# Do not fail for deprecation warnings: TaskExecutionGraph.beforeTask|afterTask
```

Comment:
❓ Do we already have an issue about removing `beforeTask` and `afterTask`? We should have one and include a note about removing these samples as well.

Reply:
There is this: https://github.com/gradle/gradle/issues/22906